core-ui/src/components/Predefined/Details/Pod/ContainerStatus.js

```diff
@@ -36,9 +38,9 @@ export function ContainerStatus({ status }) {
         i18n={i18n}
         resourceKind="containers"
         additionalContent={message}
-        type={badgeType(state.reason || 'Running')}
+        type={badgeType(state?.reason || 'Unknown')}
```

Comment:
I guess this should not be `Unknown` but `running` || `waiting` || `terminated` based on the object it comes from `status?.state?.running ||
    status?.state?.waiting ||
    status?.state?.terminated;`